### ISSUE TITLE
[SI-REVIEW] 1.1.1 Non-text Content - Descriptive 'aria-labelledby' attribute

### ISSUE THREAD

**Issue by cathysnider** [CuBoulder/express#960]
## Problem Description per Siteimprove

This warning will appear for all of your 'aria-labelledby' since a manual review is needed. WAI-ARIA 'aria-labelledby' attributes should be descriptive of the elements they are labelling, even if shown out of context.

## Page Description

- An FAQ page with 8 questions. Each answer generated this notice.
- A Quicktab accordion set
- A Quicktab tab set

## What we're doing: 

**FAQ:**
Each question is given an ID; the corresponding answers are aria-labelledby that ID. This is correct behavior. 
Example (simplified):
< div id="ui-accordion-1-header-0" aria-controls="ui-accordion-1-panel-0" >Question One: Is it spring yet?< / div >
< div id="ui-accordion-1-panel-0" aria-labelledby="ui-accordion-1-header-0" >Answer One: Yes it is.< / div >

**QuickTab:**
Basically same as above with a bunch more divs thrown in.

## Solution

Nothing to fix; the FAQs and Quicktabs are coded correctly.
